Document "Reference/Commands/vcpkg  x-add-version" (#128)
microsoft/vcpkg-docs: vcpkg/users/versioning.md

### <a name="port-version"></a> `port-version`

> Port-versions in vcpkg allow for differentiation between changes in the vcpkg-specific configurations and changes in the source package itself.

Reviewer: ```suggestion
Port-versions track changes in the packaging files (`vcpkg.json`, `portfile.cmake`, etc) without any changes to the upstream library version.

```

> A port version is a positive integer value that increases each time the port changes without updating the sources.

Reviewer: ```suggestion
A port version is a non-negative integer value.

```

- `2021-01-01#20` < `2021-01-01.1`
- `windows#7` < `windows#8`

[!NOTE]

> When making changes to a port, it's important to ensure that the [version database](versioning.concepts.md#acquiring-port-versions) is updated. To facilitate this, you can use the [`vcpkg x-add-version`](..\commands\add-version.md) command.

Reviewer: I don't think the `port-versions` section of the Versioning Reference is the right place for this.
